alchemist/src/orchid/resources/wiki/usage/indoor.md

```diff
@@ -4,4 +4,16 @@ title: "Simulation of indoor environments"
 
 ---
 
+In order to load the map of the indoor environment, you can use the {{ anchor('ImageEnvironment') }}.
 
+ImageEnvironment loads the map as raster image from file, interpreting the black pixels as obstacles
```

Comment:
Actually the color can be choosen. Can you update?

Reply:
resolved